src/Processors/Formats/Impl/ArrowColumnToCHColumn.cpp

```diff
@@ -1307,6 +1307,16 @@ static ColumnWithTypeAndName readNonNullableColumnFromArrowColumn(
                 auto tmp_lc_column = lc_type->createColumn();
                 auto tmp_dict_column = IColumn::mutate(assert_cast<ColumnLowCardinality *>(tmp_lc_column.get())->getDictionaryPtr());
                 dynamic_cast<IColumnUnique *>(tmp_dict_column.get())->uniqueInsertRangeFrom(*dict_column.column, 0, dict_column.column->size());
+                size_t expected_dictionary_size = dict_column.column->size() + (dict_info.default_value_index == -1) + is_lc_nullable;
```

Comment:
For some reason tidy build complains about `use of undeclared identifier 'is_lc_nullable'`, no idea why

Reply:
Funny enough that I don't have this or anything similar on amd64
https://pastila.nl/?001f571a/62ea190be7b7322e30081eafaff0e4f9#Y2pvpfpE9Dochz7ABkXT5w==